mlir-sycl/lib/Dialect/IR/SYCLOpsTypes.cpp

```diff
@@ -143,6 +143,32 @@ void mlir::sycl::printAccessAddrSpace(AsmPrinter &Printer,
   Printer << accessAddressSpaceAsString(AccAddress);
 }
 
+std::string
+mlir::sycl::decAccessAsString(mlir::sycl::DecoratedAccess DecAccess) {
```

Comment:
NIT: I'd use the full `decoratedAccessAsString` name here as in the rest of the cases, just for consistency.